src/Entities/User.php

```diff
@@ -28,18 +28,59 @@
 class User extends Entity
 {
     /**
-     * Try mention
+     * tryMention
      *
-     * @return string|null
+     * @param bool $markdown
+     *
+     * @return string
      */
-    public function tryMention()
+    public function tryMention($markdown = false)
     {
-        if ($this->username === null) {
-            if ($this->last_name !== null) {
-                return $this->first_name . ' ' . $this->last_name;
+        if (isset($this->username)) {
```

Comment:
Instead of using `$this->whatever`, the way it works no is to use `$this->getProperty('whatever')`, as that checks if the value exists / is `null`.

To make it easier, just set `$username = $this->getProperty('username');`, then use the `$username` variable. Same applies for other properties.